README.md

```diff
@@ -78,7 +78,7 @@ The best way to get started is to follow the [tutorial](https://docs.transform.c
 mf tutorial
 ```
 
-There are several examples of MetricFlow configs on common data sets in the [config-templates](/Users/nicholashandel/repositories/metricflow/config-templates) folder. The tutorial will rely on a small set of [sample configs](/Users/nicholashandel/repositories/metricflow/metricflow/cli/sample_models).
+There are several examples of MetricFlow configs on common data sets in the [config-templates](./config-templates) folder. The tutorial will rely on a small set of [sample configs](./metricflow/cli/sample_models).
```

Comment:
These get copied to the PyPi package description, so they have to be fully resolvable from outside github. I'm not 100% sure how best to do this for a directory, but I think this will work:

```suggestion
There are several examples of MetricFlow configs on common data sets in the [config-templates](https://github.com/transform-data/metricflow/tree/main/config-templates) folder. The tutorial will rely on a small set of [sample configs](https://github.com/transform-data/metricflow/tree/main/metricflow/cli/sample_models).
```